src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IDelegateCreationExpression.cs

```diff
@@ -0,0 +1,976 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void DelegateCreationExpression_ImplicitLambdaConversion()
+        {
+            string source = @"
+using System;
+class Program
+{
+    void Main()
+    {
+        /*<bind>*/Action a = () => { };/*</bind>*/
+    }
+}
+";
+            string expectedOperationTree = @"
+IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'Action a = () => { };')
+  IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'Action a = () => { };')
+    Variables: Local_1: System.Action a
+    Initializer: IDelegateCreationExpression (OperationKind.DelegateCreationExpression, Type: System.Action) (Syntax: '() => { }')
```

Comment:
>Initializer: IDelegateCreationExpression (OperationKind.DelegateCreationExpression, Type: System.Action) (Syntax: '() => { }') [](start = 4, length = 126)

Should this node be Implicit.

Reply:
It is. Added a separate test that attempts to GetOperation just for the lambda expression to verify that we're only seeing the IAnonymousFunctionExpression node, not the wrapping IDelegateCreationExpression.

---
In reply to: [138728156](https://github.com/dotnet/roslyn/pull/22056#discussion_r138728156) [](ancestors = 138728156)